CHANGELOG.md

```diff
@@ -6,6 +6,7 @@
 ### Added
 
 ### Changed
+* [XENOPS-891](https://xenitsupport.jira.com/browse/XENOPS-891): update docker images and wars to latest hotfix for 52, 62 & 70
```

Comment:
I've only seen this now because I'm releasing, but please only add tickets from ALFREDAPI to this repo. In this case, create a new product ticket and use that.